src/include/log.h

```diff
@@ -38,6 +38,20 @@ mender_err_t mender_log_init(void);
  */
 mender_err_t mender_log_exit(void);
 
+#ifdef CONFIG_MENDER_DEPLOYMENT_LOGS
+/**
+ * @brief Activate deployment logs gathering
```

Comment:
General comment,

I think "store" or "save" is better than "gather". (If you decide to change, do so both in the docs and in the log messages)